Jenkinsfile

```diff
@@ -1,20 +1,29 @@
 #!groovy
 
+def review(self,labels, message) {
+  try {
+    self.gerritReview labels: labels, message: message
+  } catch (NoSuchMethodError e) {
+  }
+}
+
 // Don't test plugin compatibility - exceeds 1 hour timeout
 // Allow failing tests to retry execution
 // buildPlugin(failFast: false)
 
 // Test plugin compatbility to latest Jenkins LTS
 // Allow failing tests to retry execution
 try {
-  gerritReview labels: [:], message: "Build started ${env.BUILD_URL}"
+  review self:this, labels: [:], message: "Build started ${env.BUILD_URL}"
```

Comment:
I would have considered to check if this self:this contains gerritReview via bindings and if not add add a dummy object.